platforms/core-configuration/base-diagnostics/src/testFixtures/groovy/org/gradle/api/reporting/model/ModelReportParser.groovy

```diff
@@ -144,7 +145,7 @@ class ModelReportParser {
     }
 
     private static List<String> nodeOnlyLines(List<String> nodeLines) {
-        int successMarker = nodeLines.findIndexOf { line -> line.startsWith(END_OF_REPORT_MARKER) }
+        int successMarker = nodeLines.findIndexOf { line -> line.find(END_OF_REPORT_MARKER) }
```

Comment:
💭 This isn't strictly equivalent, but the model report is on its way out anyway.